google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -101,6 +101,21 @@ public BucketInfo apply(com.google.api.services.storage.model.Bucket pb) {
   private final String locationType;
   private final Logging logging;
 
+  public enum PublicAccessPrevention {
```

Comment:
Do we have some docs we can point to to provide some context for why this enum exists? Maybe something like is on `IamConfiguration` below.